doc/source/cluster/kuberay.md

```diff
@@ -1,116 +1,11 @@
-# Deploying with KubeRay
+(kuberay-index)=
```

Comment:
Re-purposing this as the index page and retitling from `Deploying with KubeRay` to `Ray on Kubernetes`.